x/concentrated-liquidity/incentives_test.go

```diff
@@ -2563,17 +2587,15 @@ func (s *KeeperTestSuite) TestCreateIncentive() {
 					incentiveRecordOne.IncentiveRecordBody.RemainingAmount.Ceil().RoundInt(),
 				),
 			),
-			recordToSet:       withMinUptime(incentiveRecordOne, time.Hour*3),
-			authorizedUptimes: types.SupportedUptimes,
+			recordToSet: withMinUptime(incentiveRecordOne, time.Hour*3),
 
 			expectedError: types.InvalidMinUptimeError{PoolId: 1, MinUptime: time.Hour * 3, AuthorizedUptimes: types.SupportedUptimes},
 		},
 		"insufficient sender balance": {
-			poolId:            defaultPoolId,
-			sender:            sdk.MustAccAddressFromBech32(incentiveRecordOne.IncentiveCreatorAddr),
-			senderBalance:     sdk.NewCoins(),
-			recordToSet:       incentiveRecordOne,
-			authorizedUptimes: types.SupportedUptimes,
```

Comment:
Dont we want at least one test where the only authorizedUptime is the 1ns uptime, just to sanity check that this does not produce any unexpected behaviors?

Reply:
Hmm yeah this is a good point -- normally that was the default case, so maybe switching to a different default than what we're launching with isn't the best move.

Considering reverting that part of this refactor (switching to draft for now)

Reply:
Ended up reverting the authorized uptime part of the refactor as in hindsight it's not clear to me that it's a definite win. Would be cleaner to do test multiple authorized uptime permutations on all incentives tests instead